dashboards/ebs/ebs_snapshot_detail.sp

```diff
@@ -136,7 +153,7 @@ query "aws_ebs_snapshot_relationships" {
     from
       snapshot
 
-    -- EBS - nodes
+    -- From EC2 EBS (node)
```

Comment:
```suggestion
    -- From EBS volumes (node)
```

Reply:
Please follow the format of `-- To/From <service> <resource plural> (node)` for nodes and `-- To/From <service> <resource plural> (edge)` for edges